### PR TITLE
Remove unused class `TxHelper`

### PR DESCRIPTION
**Pull request by barakman**

### CODE REVIEW

**Review by sklbancor — CHANGES_REQUESTED** [review]
Rather than deleting unused files one by one can we please move them to resources/deleted and then delete them all at once? sometimes there is code that you want to review / reuse and it can be hard to find when you don't know exactly where it was or when it was deleted.

File: fastlane_bot/utils.py [hunk 1]
```diff
@@ -282,15 +282,3 @@ def find_latest_timestamped_folder(logging_path=None):
 
     list_of_folders.sort(reverse=True)  # Sort the folders in descending order
     return list_of_folders[0]  # The first one is the latest
-
-
-def count_bytes(data: HexBytes) -> (int, int):
```
Comment (sklbancor) [inline fastlane_bot/utils.py:287]: just pointing out that this is not the docstring format we want to use in the main library

**Review by sklbancor — DISMISSED** [review]

File: fastlane_bot/helpers/txhelpers.py [hunk 2]
```diff
@@ -15,145 +15,25 @@
 # import random
 # import time
 from dataclasses import dataclass
-from typing import List, Any, Dict, Tuple, Optional
+from typing import List, Any, Dict, Optional
```
[the remark this answers is not in the document]
Comment (barakman) [reply fastlane_bot/helpers/txhelpers.py:18]: Will handle this separately, in order to keep the changes strictly related to this PR

**Comment by barakman** [comment]
> Collaborator

Will handle this separately, in order to keep the changes strictly related to this PR.

**Comment by barakman** [comment]
> Rather than deleting unused files one by one can we please move them to resources/deleted and then delete them all at once? sometimes there is code that you want to review / reuse and it can be hard to find when you don't know exactly where it was or when it was deleted.

Will handle this separately, in order to keep the changes strictly related to this PR.

**Review by zavelevsky — APPROVED** [review]
LGTM

**Pull request merged**